src/actions/sso/types.ts

```diff
@@ -29,6 +30,7 @@ export interface PaymentRequestSummary extends RequestSummary {
     did: string
     address: string
   }
+  requestJWT: string
```

Comment:
Ideally we get rid of this entirely, since it does not belong on this layer. This is still progress because it's no longer on the `RequestSummary` interface.

Reply:
as for payment interactions, they were not added to the InteractionManager, so the requestJWT is still required on that interface. (if I understood your comment correctly)

Reply:
It's not present on the `RequestSummary` interface though, only on the PaymentRequestSummary, which extends it.